common/source/docs/common-rangefinder-nooploop-tofsense-p.rst

```diff
@@ -1,13 +1,13 @@
 .. _common-rangefinder-nooploop-tofsense-p:
 
 ===================
-Nooploop TOFSense P
+Nooploop TOFSense/-UART/S
```

Comment:
under/overline

Reply:
Sorry, I didn't quite understand what you meant. Our product name is TOFSense-UART. Do you mean we need to change/- UART to/UART here?

Reply:
no, the row of "=" above and below titles must be the same length as the text of the tiltle

Reply:
Okay, thank you. It's fixed

Reply:
looks good...I squashed for you...will merge once CI passes again